.github/CODEOWNERS

```diff
@@ -4,4 +4,4 @@
 # the repo. Unless a later match takes precedence,
 # these owners will be requested for review when someone
 # opens a pull request.
-*       @dav3r @felddy @jsf9k @mcdonnnj @cisagov/team-ois
+*       @dav3r @hillaryj @mcdonnnj
```

Comment:
We are removing the extra spaces before the first name in our list of code owners now.  When we first started adding CODEOWNERS files, we thought those spaces were a good idea for some reason. 🤷‍♂️

Reply:
Providing context for why, the [original blog article](https://github.blog/2017-07-06-introducing-code-owners/) announcing CODEOWNERS has that spacing in the example. I think it was to keep the list of codeowners even down the file, but on its own it doesn't make much sense. I copied the example from the article below in case you don't want to follow the link:

```
# Lines starting with '#' are comments.
# Each line is a file pattern followed by one or more owners.

# These owners will be the default owners for everything in the repo.
*       @defunkt

# Order is important. The last matching pattern has the most precedence.
# So if a pull request only touches javascript files, only these owners
# will be requested to review.
*.js    @octocat @github/js

# You can also use email addresses if you prefer.
docs/*  docs@example.com
```

Reply:
Removed spaces in https://github.com/cisagov/assessment-data-import-lambda/pull/6/commits/6b43a8120cda6bd7a4b7286e04e729407ec238c0